templates/galaxy/config/job_conf.xml.j2

```diff
@@ -1,101 +1,139 @@
 <job_conf>
 	<plugins workers="4">
 		<plugin id="local_plugin" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner"/>
+
 		<plugin id="condor" type="runner" load="galaxy.jobs.runners.condor:CondorJobRunner"/>
 		<plugin id="pulsar_embedded" type="runner" load="galaxy.jobs.runners.pulsar:PulsarEmbeddedJobRunner">
 			<param id="pulsar_config">{{ galaxy_config_dir }}/pulsar_app.yml</param>
 		</plugin>
+		<plugin id="pulsar_runner" type="runner" load="galaxy.jobs.runners.pulsar:PulsarMQJobRunner" >
```

Comment:
so we'll need three plugins here, one PER queue we want to talk to, with the appropriate user/vhost/password templated out.. (Many bigger galaxies template theirs in a loop or so, but let's start with just pasted versions

Reply:
well lets start good, i made it into a forloop (*also as personal exercise)

Reply:
sounds good!